markdown/preface.markdown

```diff
@@ -11,69 +11,76 @@ Elegant SciPy goes further.
-We hope that you will know exactly what he means after reading this book.
+Most would use it to describe the visual simplicity, style, and grace of, say, the first iPhone.
+Instead Graeme Simsion's hero, Don Tillman, uses a computer algorithm to *define* elegance.
+We hope that you will understand exactly what he means after reading this book.
 That you will read or write a piece of elegant code, and feel calmed in the glow of its beauty and grace.
 (Note: the authors may be prone to hyperbole.)
 Perhaps you too will compliment your significant other on how their outfit reminds you of that bit of code you saw in Elegant SciPy...
 
 So, what makes code elegant?
 Elegant code is a pleasure to read, use and understand because it is:
 
-* Simple
+* Concise
 * Efficient
 * Clear
 * Creative
 
-Elegant code achieves much in a few lines, through abstraction and functions, *not* through just packing in a bunch of nested function calls!
-Elegant code is often efficient, not only in number of keystrokes but also in time and memory.
-Elegant code should be clear and easy to understand.
-It may take a minute to grok, but it should provide a crisp moment of understanding.
-This can be done through clear variable and function names, and carefully crafted comments, explaining not what is being done (the code does this), but *why* it is being done.
-
-Writing this simple, efficient, clear code requires significant creativity.
-In the New York Times, software engineer J. Bradford Hipps [recently argued](http://www.nytimes.com/2016/05/22/opinion/sunday/to-write-software-read-novels.html) that "to write better code, [one should] read Virginia Woolf."
-You might use a particularly efficient data structure in a new, unexpected context.
-
-In many cases elegant code intrigues us because it does something clever, approaching a problem in a new way, or just in a way that in retrospect is obvious in its simplicity.
-It is the culmination of these elements of elegant code that make your code "beautiful", a pleasure to write, read and to use.
-This is elegant code.
-
-Now that we’ve dealt with the elegant part of the title, let’s bring back the SciPy.
-
-SciPy is a library, an ecosystem, and a community.  Part of what makes
-these great is that they have excellent online documentation and
+The conciseness of elegant code comes through abstraction and functions, *not* just through packing in a bunch of nested function calls!
+It may take a minute or two to grok, but it should ultimately provide a crisp, "ah-ha!" moment of understanding.
+Once you know the various components of the code, its correctness should be
+obvious!
+This can be aided by clear variable and function names, and carefully crafted comments that *explain* the code, rather than merely *describe* it.
+
+Creativity should support the first three goals.
+Creativity for its own sake can lead to obtuse code that is hard to understand.
+Make sure you are not too clever for your readers' sake!
```

Comment:
Odd phrase. What about something like you are showing off at the expense of your reader's understanding?